.github/workflows/nodejs.yml

```diff
@@ -23,5 +27,21 @@ jobs:
           cache: npm
       - name: Install
         run: npm ci
+      - name: Node version
+        id: version
+        run: echo "node-version=$(node -v)" >> $GITHUB_OUTPUT
       - name: Test
+        id: test
         run: npm test
+        continue-on-error: ${{ matrix.node-version == 'current' }}
+      - name: Create Issue on Fail
```

Comment:
Is this going to create a new issue on every failure?

Reply:
Yes it will. Can you think of a way to improve that?